.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 .vscode
+junit.xml
```

Comment:
When running tests locally, this file pops up. Think we may ignore it.

Reply:
This is for CI only.